### PR TITLE
Add al_MASS's example for Sudoku in Examples/Sudoku_al_MASS.md

### PR DESCRIPTION
**Pull request by sreetamdas**

### CODE REVIEW

File: Examples/Sudoku_al_MASS.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Sudoku
+
+Documentation for the Problem Statement by Team al_MASS during Hackathon 5.0
+<br><br>
+<b>Introduction:</b>
```
Comment (sudheesh001) [inline Examples/Sudoku_al_MASS.md]: Replace this with `### Introduction` or in markdown its written as `__Introduction__` for rendering it as **Introduction**

File: Examples/Sudoku_al_MASS.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<b>Requirements:</b>
+Computer with MATLAB, ADB Tool and required drivers set up. An Android Device with the ‘Sudoku Free’ game installed on it. (Turn on the Developer options for better visualization) Keep a USB data transfer cable handy.
+<br><br><br><br>
+<b>Block Diagram:</b><br>
+![alt tag](https://raw.githubusercontent.com/sreetamdas/al_MASS/master/doc1.png)
+<br><br><br><br>
+<b>Tutorial</b><br><br>
+<b>Step 1: Using ADB Tool to capture screenshot</b>
+The following command instantaneously takes the screenshot of the connected device and stores it in the SD card following the specified path.<br><br>
+`system(' adb shell screencap -p /sdcard/screen.png ');`<br><br>
```
Comment (sudheesh001) [inline Examples/Sudoku_al_MASS.md]: Add spacing between each paragraph, replace from HTML to pure markdown.

**Comment by sudheesh001** [comment]
@sreetamdas Good work, but please replace HTML with Markdown, http://dillinger.io/ would be of good help to get you started on markdown.

**Comment by sreetamdas** [comment]
Add Documentation for **Simon Says** in Examples/

File: Examples/SimonSays.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Simon Says
+Documenation for the game __Simon Says__, solved by __Team al_MASS__ for Hackathon 5.0
```
Comment (sudheesh001) [inline Examples/SimonSays.md:3]: Maintain consistency in both files, put a new line after `# Simon Says`

**Comment by sreetamdas** [comment]
Any other edits?

**Comment by sudheesh001** [comment]
Looks good !

**Pull request merged**